MetaMask/metamask-extension
# Fix #22940 - Remove inline gas components from new send flow

## **Description**

Removes the inline gas components from the new send flow, as they have also been removed from the present send flow.

## **Related issues**

Fixes: #22936

## **Manual testing steps**

1. `MULTICHAIN=1 yarn start`
2. Click "send"
3. Select a recipient
4. *Don't* see the old inline gas components.

## **Screenshots/Recordings**



### **Before**

<img width="236" alt="SCR-20240214-jmuk" src="https://github.com/MetaMask/metamask-extension/assets/46655/2eedaded-4d28-4d9d-b0d3-8e9771beb480">

### **After**

<img width="425" alt="SCR-20240214-jnjf" src="https://github.com/MetaMask/metamask-extension/assets/46655/fb73d2f6-6541-4c07-849f-ca85b199aa32">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
